### DISCUSSION TITLE
wezterm in terminal path

### DISCUSSION THREAD

**Discussion by jutbin** [wezterm/wezterm#6091] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Why is it that when I use the latest version of wezterm and type wezterm-gui in the cmd of Windows 10, the interface that pops up is not the path in my cmd but the default path? Can't I call it from the path in cmd?

**Comment by bew** [comment]
You'll need to tell it to start in another folder if you want that.

See the decision diagram in this page for more information:
https://wezfurlong.org/wezterm/config/lua/config/default_cwd.html
